app/src/main/java/com/zulip/android/models/Message.java

```diff
@@ -198,6 +201,7 @@ public Message(ZulipApp app, JSONObject message,
 
         this.setTimestamp(new Date(message.getLong("timestamp") * 1000));
         this.setMessageRead(false);
+        this.setMessageStar(false);
```

Comment:
@kunall17 i agree, I think we first would need to add a read field in messages list for json response